src/matter/cluster/GroupKeyManagementCluster.ts

```diff
@@ -0,0 +1,135 @@
+const TlvGroupInfoMapStruct = TlvObject({ 
+    groupId: TlvField(1, TlvGroupId), /* min: 1 */
+    endPoints: TlvField(2, TlvArray(TlvEndpointNumber)),
+    groupName: TlvField(3, TlvString.bound( { maxLength: 16 })),
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.6.2 table 87 */
+export const enum GroupKeySecurityPolicyEnum {
+    /** Message counter synchronization using trust-first */
+    TrustFirst = 0x00,
+  
+    /** Message counter synchronization using cache-and-sync */
+    CacheAndSync = 0x01,
+  }
+
+  /** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.1 */
+const TlvKeySetWriteRequest= TlvObject({
+    groupKeySet: TlvField(0, TlvGroupKeySetStruct, )
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.1 */
+const TlvKeySetWriteResponse= TlvObject({
+    status: TlvField(0, TlvEnum<StatusCode>()), 
+});
+
+  /** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.2 */
+const TlvKeySetReadRequest= TlvObject({
+    groupKeySetId: TlvField(0, TlvUInt16, )
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.3 */
+const TlvKeySetReadResponse= TlvObject({
+    groupKeySet: TlvField(0, TlvGroupKeySetStruct),
+});
+
+  /** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.4 */
+const TlvKeySetRemoveRequest= TlvObject({
+    groupKeySetId: TlvField(0, TlvUInt16, )
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.4 */
+const TlvKeySetRemoveResponse= TlvObject({
+    status: TlvField(0, TlvEnum<StatusCode>()), //TODO
+});
+
+ /** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.9.5 */
+const TlvKeySetReadAllIndicesRequest= TlvObject({
+    // Spec says no payload which seems correct but chip XML shows the following
+    // groupKeySetIds: TlvField(0, TlvArray(TlvUInt16), )
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} §1 1.2.9.6 */
+const TlvKeySetReadAllIndicesResponse= TlvObject({
+    groupKeySetIds: TlvField(0, TlvArray(TlvUInt16), )
+});
+
+/**
+ * The Group Key Management Cluster is the mechanism by which group keys are managed.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2
+ */
+export const GroupKeyManagementCluster = Cluster({
+    id: 0x3f,
+    name: "GroupKeyManagement",
+    revision: 1,
+    features: {
+        /** The ability to support CacheAndSync security policy and MCSP. */
+        cacheAndSync: BitFlag(0),  
+    },
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.7.1 */
+    attributes: { 
+        /** Each entry associates a logical Group Id with a particular group key set. */
+        groupKeyMap: WritableAttribute(0, TlvArray(TlvGroupKeyMapStruct, { maxLength: 254 })),
+
+        /** Each entry provides read-only information about how a given logical Group ID maps to a particular set of endpoints */
+        groupTable: Attribute(1, TlvArray(TlvGroupInfoMapStruct, { maxLength: 254 })),
```

Comment:
And please add /* fabricSensitive: true */ at line end (This is the "F" in the Access column)